_opensearch/query-dsl/text-analyzers.md

```diff
@@ -48,6 +48,12 @@ Each analyzer consists of one tokenizer and zero or more token filters. Differen
   }
   ```
 
+## Analyzer options
+
+Option | Valid values | Description
+:--- | :--- | :---
+`analyzer` | `standard, simple, whitespace, stop, keyword, pattern, language, fingerprint` | The analyzer you want to use for the query. Different analyzers have different character filters, tokenizers, and token filters. The `stop` analyzer, for example, removes stop words (e.g., "an," "but," "this") from the query string. For a full list of acceptable language values, see [Convert text with analyzers](#convert-text-with-analyzers) on this page.
```

Comment:
"... removes stop words (for example, "an," "but," "this") ..."

Reply:
The link is broken here - there is no convert text with analyzers section in this file.

Reply:
There is no [Convert text with analyzers](#convert-text-with-analyzers) on the text analyzers page.